README.md

```diff
@@ -121,6 +121,19 @@ injector().inject(function(TasksPrinter){
   TasksPrinter.print();
 });
 ```
+
+##### Usage note for ES6 syntax
+
+While it is tempting to utilize the fat arrow syntax in this top declaration file like the example below, it will not be supported by spur-ioc. For more information, read issue [#26](https://github.com/opentable/spur-ioc/issues/26). Instead use the recommended approach above. There isn't a compelling reason to add that additional support. If you use this style, it will break as the report in issue #26.
+
+```javascript
+var injector = require("./injector");
```

Comment:
Only for the function definition. Imports are not supported in most versions of node, so I kept it as close to what is supported on most versions.